src/components/CCIP/Landing/NetworkGrid.tsx

```diff
@@ -18,6 +18,22 @@ const BEFORE_SEE_MORE = 2 * 7 // Number of networks to show before the "See more
 
 function NetworkGrid({ networks, environment }: NetworkGridProps) {
   const [seeMore, setSeeMore] = useState(networks.length <= BEFORE_SEE_MORE)
+
+  useEffect(() => {
+    const urlParams = new URLSearchParams(window.location.search)
+    if (urlParams.get("showAll") === "true") {
+      setSeeMore(true)
+    }
+  }, [])
+
+  const handleSeeMoreClick = () => {
+    setSeeMore(true)
+    const urlParams = new URLSearchParams(window.location.search)
+    urlParams.set("showAll", "true")
+    const newUrl = `${window.location.pathname}?${urlParams.toString()}`
+    window.history.replaceState({ path: newUrl }, "", newUrl)
+  }
```

Comment:
Three points:
- I do not think we should show the parameter if someone clicks the show all manually, it is good to keep it hidden
- Add a comment on the "seeMore" logic, so that people know why it has been added (algolia crawler)
- Auto "show" also all the tokens in this page